swagger/v4/swagger.json

```diff
@@ -1176,6 +1188,18 @@
         "id",
         "password"
       ]
+    },
+    "app_user_capabilities": {
+      "type": "object",
+      "properties": {
+        "can_teleconsult": {
+          "type": "string",
+          "enum": [
+            "yes",
+            "no"
+          ]
+        }
+      }
```

Comment:
Will we update the `FacilityTeleconsultationsController` after the data migration is complete? I guess that's the last step?

Reply:
The plan is to leave this controller untouched. Any nurses who wish to create new style teleconsults will have to update their apps anyways, and we expect this endpoint to stop being used after a while.

Reply:
Oh, we do? I thought we'd replace the contents with MOs through the new data model? That way nurses don't _have_ to update right?

PS, how is the new version of the app planning to fetch telemed numbers from the server?

Reply:
The new app will fetch the list from `teleconsultation_medical_officers/sync` and populate the bottom sheet with that data. We _could_ serialize the MOs in this old API but not sure if it's worth the effort. It's likely that all teleconsult nurses will be asked to update to be able to send that new deeplink message.

Reply:
Understood. Nice.